mock-backend/src/controllers/productController.js

```diff
@@ -1,7 +1,24 @@
 const products = require("../data/mokedData");
 
+const validateNumberQueryParam = (value, defaultValue = 1) => {
+  return value && !Number.isNaN(value) ? Number(value) : defaultValue;
```

Comment:
Did you try to use just isNaN? It automatically tries to coerse value to number, but I am not sure about typescript. You should try it. Then you don't need to check for value

Reply:
Thanks, I used `isNaN`, it looks better now